tests/test_enterprise/views/test_router_view.py

```diff
@@ -215,8 +225,19 @@ def test_get_redirects_with_course_key_and_sessionToken(
             'course_key': 'fake_course_key',
         }
         self.request.GET['sessionToken'] = 'fake_token'
+        self.request.GET['subdomain'] = 'dummy_subdomain'
```

Comment:
since this is required to be contained by the base url in the test config defined in the setup, would it be smart to keep it as a constant, perhaps as a class var? then you can construct the base url with said constant